setup.py

```diff
@@ -25,7 +25,7 @@
          ]
     },
     include_package_data=True,
-    install_requires=['pandas>=1.0', 'xlrd==1.2.0'],
+    install_requires=['pandas>=1.0', 'openpyxl>=3.0.6'],
```

Comment:
Perhaps worth a comment in the README about what Excel file types are and aren't supported?

Reply:
I can do that, but really all Excel files are coming from the UAS, so in theory, should be the same.

Reply:
Ok, well probably not a concern then.